README.md

```diff
@@ -12,87 +12,154 @@
 
 # TestResultWebApp (TRWebApp)
 
-::: {.contents local=""}
-:::
+## Table of Contents
+
+-   [Getting Started](#getting-started)
+    -   [How to install](#how-to-install)
+-   [Usage](#usage)
```

Comment:
Hi Ngoan,
a chapter how to upload data is missing.
RobotLog2DB...
Thank you,
 Thomas